app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -91,5 +91,33 @@
       </div>
       </fieldset>
     </div>
+    <!--advanced options-->
+    <div>
+      <a href=""
+         ng-click="showAdvancedOptions = true"
+         ng-show="!showAdvancedOptions">Show Advanced Options for Requested Storage</a>
```

Comment:
Since we're only showing label controls, let's just say that. Also we use sentence case for these links. Maybe

"Use label selectors in the storage request."

Probably want a "Learn more" link to this topic: https://docs.openshift.org/latest/install_config/persistent_storage/selector_label_binding.html